src/dct/Region.lua

```diff
@@ -212,6 +212,20 @@ function Region:_generate(assetmgr, objtype, names)
 		limits.limit = math.random(limits.min, limits.max)
 	end
 
+	for i, tpl in pairs(names) do
```

Comment:
```suggestion
	for i, tpl in ipairs(names) do
```

You want to use `ipairs` since names is an 'array'.